### DISCUSSION TITLE
What is the minimum Digital Ocean droplet for Dokku to go through the tutorial?

### DISCUSSION THREAD

**Discussion by NathanWailes** [dokku/dokku#7099] [Q&A] [answered]
I saw in the Dokku docs that I needed 1 GB of RAM, but when I then went through the Dokku "Deploying an Application" guide, `git push dokku main` would hang, and I saw [this discussion](https://github.com/dokku/dokku/discussions/4930) saying that it was likely an out-of-memory problem.

So do I use a 2 GB RAM droplet to go through the tutorial and then downgrade to 1GB if I'm just using Go as my backend?

**Comment by NathanWailes** [comment, marked as answer]
Yes, that seems to be what I should be doing. git push dokku main worked after I upgraded to a $12/mo 2 GB RAM droplet. So I'll use the 2 GB RAM droplet to get the demo app working and then try downgrading to the 1 GB RAM droplet when I switch to trying to get Dokku working with my Go backend app.

**Reply by josegonzalez** [reply]
You can also just add swap memory (we have docs on how to do this on our site). It's slower but should work fine for lower-power servers.
